tests/Unit/Evolution/Systems/GrMhd/ValenciaDivClean/BoundaryConditions/Test_Reflective.cpp

```diff
@@ -37,10 +37,15 @@ void test_stuffs(const bool reflect_both) {
           make_not_null(&gen), used_for_size);
   const auto sqrt_det_spatial_metric = determinant(spatial_metric);
 
+  struct RandomBool : db::SimpleTag {
```

Comment:
Maybe name this `ReflectBoth` instead of `RandomBool`? "random" means something in CS, and it does not appear the value of this bool is randomly generated

Reply:
@nilsdeppe done.